components/breadcrumb/StarmapsBreadcrumbItem.tsx

```diff
@@ -0,0 +1,32 @@
+import { BreadcrumbItem, BreadcrumbItemProps, BreadcrumbLink } from '@chakra-ui/react';
+import React from 'react';
+import { useGlobalLoadingState } from '../../hooks/useGlobalLoadingState';
+
+interface StarmapsBreadcrumbItemProps extends BreadcrumbItemProps {
+  title: string;
+  url: string;
+}
+
+export function StarmapsBreadcrumbItem({ title, url, ...props }: StarmapsBreadcrumbItemProps) {
+  const globalLoadingState = useGlobalLoadingState();
+
+  const breadcrumbItemProps = {
+    ...props,
+    color: '#4987BD',
+    onClick: () => {
+      globalLoadingState.start();
+      setTimeout(() => globalLoadingState.stop(), 5000);
+    },
+    className: 'js-breadcrumbItem js-breadcrumbItem-link',
```

Comment:
Js-prefix classNames are specifically for grabbing with js. Adding them in a modules.css file with empty styling (because they shouldn't have styles) doesnt make sense.

Btw, i need to document this in a code patterns guide for our team somewhere